pkg/apis/resource/types.go

```diff
@@ -232,6 +370,17 @@ type DeviceCapacity struct {
 	// capacity (= share a single device between different consumers).
 }
 
+// Counter describes a quantity associated with a device.
+type Counter struct {
+	// Value defines how much of a certain device counter is available.
+	//
+	// +required
+	Value resource.Quantity
+
+	// potential future addition: fields which define how to "consume"
```

Comment:
remove this comment - this applies to DeviceCapacity but NOT Counter

if you want a potential future addition, it would be an IntRange type that allows defining ranges of integers rather than creating separate counters for each integer